src/v/kafka/server/handlers/topics/types.h

```diff
@@ -58,6 +58,30 @@ static constexpr std::string_view topic_property_data_policy_function_name
 static constexpr std::string_view topic_property_data_policy_script_name
   = "redpanda.datapolicy.script.name";
 
+// Kafka topic properties that is not relevant for Redpanda
+static constexpr std::string_view allowlist_topic_noop_confs[3] = {
```

Comment:
Seems that we are hardcoding the list here, even if Kafka doesn't add new topic configurations frequently, shouldn't we future proof by making this list accessible via redpanda.yaml on the broker?

Reply:
I think they really won't add new configurations often. Even if they add, it will take a time for users to start using them, and if they later want to switch to redpanda with them, we will already add them here.

Otherwise, adding it to config will increase config size with not clear properties

Reply:
Ok, lets leave the list harcoded for now then. Thanks!